distributed/distributed.yaml

```diff
@@ -72,6 +72,10 @@ distributed:
     lost-worker-timeout: 15s  # Interval after which to hard-close a lost worker job
 
   comm:
+    retry_operations:  # some operations (such as gathering data) are subject to re-tries with the below parameters
+      max_retries: 0  # the maximum number to retry an operation in case of a connection problem
+      base_delay: 1s  # the first non-zero delay between re-tries
+      max_delay: 20s  # the maximum delay between re-tries
```

Comment:
I recommend the following names:

```yaml
retry:
  count: 0
  delay:
    min: 1s
    max: 20s
```

Also, if you want to keep with multi-word names then I recommend the use of hyphens over underscores.  Mostly my reasoning here is for consistency with the names that are currently here.

Reply:
Alright, I just changed it to what you proposed. Thanks for the quick review!